src/amo/css/inc/vars.scss

```diff
@@ -18,6 +18,7 @@ $breakpoints: (
   medium: only screen and (min-width: 500px),
   large: only screen and (min-width: 720px),
   extraLarge: only screen and (min-width: 900px),
+  desktop: only screen and (min-width: 1025px),
```

Comment:
`1025px` is a pretty large desktop! My own laptop browser isn't always that wide. It seems this should fix onto an `extraLarge` display, but if not we should at least stick to the size-type naming. Something like `fullWidth` or `largest`? 🤔

Reply:
I was not used to such breakpoints to be honest 😄  (see Bootstrap ones for instance: https://getbootstrap.com/docs/3.3/css/#responsive-utilities-classes).

I was not happy with `extraExtraLarge`, hence `desktop` but I knew this would raise a red flag 😛  I am fine with not using device names.

What would be the "desktop" reference width by the way? I think 1000px is pretty common now (GitHub has 980px for example).

Reply:
Ok actually `extraLarge` is quite ok.